tests/languages/java-file/app/input.txt

```diff
@@ -0,0 +1 @@
+This is the content of file
```

Comment:
Either `This is the content of the file` or `This is the content of input.txt`.

Reply:
Fixed the grammar of data and made it two lines.